cub/cub/device/device_select.cuh

```diff
@@ -191,16 +191,17 @@ struct DeviceSelect
       SelectOp,
       EqualityOp,
       OffsetT,
-      false>::Dispatch(d_temp_storage,
-                       temp_storage_bytes,
-                       d_in,
-                       d_flags,
-                       d_out,
-                       d_num_selected_out,
-                       SelectOp(),
-                       EqualityOp(),
-                       num_items,
-                       stream);
+      /*KeepRejects*/ false,
+      /*MayAlias*/ false>::Dispatch(d_temp_storage,
```

Comment:
How about we add some `constexpr bool` definitions like `constexpr bool keep_reject = false;` instead? Same would apply for similar instantiations of the `Dispatch` template in other places.

Reply:
I agree. We already have them in the tuning policies anyway.

Reply:
Ha, except that would be a breaking change :D Will add it to the breaking change list.